src/frontend/js/components/AshleyEditor/index.tsx

```diff
@@ -121,19 +132,55 @@ export const AshleyEditor = (props: MyEditorProps) => {
     return 'not-handled';
   };
 
+  let mentionPluginRender;
+  const plugins = [toolbarPlugin, emojiPlugin, linkPlugin, codeEditorPlugin];
+
+  if (props.mention_users) {
+    const [open, setOpen] = useState(true);
+    const [suggestions, setSuggestions] = useState(props.mention_users);
+
+    const { MentionSuggestions, mentionPlugin } = useMemo(() => {
+      // tslint:disable-next-line:no-shadowed-variable
+      const mentionPlugin = createMentionPlugin();
+      // tslint:disable-next-line:no-shadowed-variable
+      const { MentionSuggestions } = mentionPlugin;
+      return { mentionPlugin, MentionSuggestions };
+    }, []);
+
+    const onOpenChange = useCallback((_open: boolean) => {
```

Comment:
You remove this callback. As the boolean is the first parameter and the only argument you pass to `setOpen` you could use `setOpen` directly.